pygotham/frontend/static/scss/_settings.scss

```diff
@@ -117,10 +117,11 @@ $white       : #FFFFFF;
 // $oil         : #333333;
 // $jet         : #222222;
 // $black       : #000000;
-$light-blue: #d0eaf4;
-$dark-blue: #182833;
-$gold: #aca034;
-$teal: #376b7a;
+// $teal: #376b7a;
```

Comment:
Since all of the others have been changed (effectively removing 75% of the 2015 colors), we might as well remove this one as well. If we need to get it back, it's in the commit history.